packages/gatsby-source-wordpress/__tests__/fetch-referenced-media-items.test.js

```diff
@@ -1,7 +1,3 @@
-/**
- * @jest-environment jsdom
```

Comment:
@wardpeet I had to remove this so the unit tests would pass again. I think this is because this test should be in a node environment, not browser, and another change caused lmdb to be imported into this test. Since lmdb can't be imported in the browser this caused the tests to fail. Removing this jsdom comment makes sense anyway and it makes the tests pass again 👍